src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/GoalResourceMapperTest.kt

```diff
@@ -47,7 +47,7 @@ internal class GoalResourceMapperTest {
     val expectedGoal = aValidGoal(
       reference = UUID.randomUUID(),
       title = createGoalRequest.title,
-      reviewDate = createGoalRequest.reviewDate,
+      reviewDate = createGoalRequest.reviewDate!!,
```

Comment:
Is this the right thing to do here?
Should the arg `reviewDate` be made optional on the `aValidGoal` function instead? 🤔

Reply:
OK 👍